libcontainer/specconv/spec_linux.go

```diff
@@ -498,6 +498,13 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 			s := *spec.Process.Scheduler
 			config.Scheduler = &s
 		}
+
+		if spec.Process.IOPriority != nil {
+			config.IOPriority = &configs.IOPriority{
+				Class:    spec.Process.IOPriority.Class,
+				Priority: spec.Process.IOPriority.Priority,
+			}
```

Comment:
This can be done in a simpler way (see the code just above). Something like

```go
p := *spec.Process.IOPriority
config.IOPriority = &p
```

Reply:
Maybe it will be possible with `type` 👍